examples/checkin_collection.sample

```diff
@@ -13,8 +13,8 @@
           "supplement": false,
           "location_id": "eb2d063a-5b4c-4cab-8db1-5fc5c5941df6",
           "material_type_id": "3cdc2cd3-6646-43bf-aacc-37934aeae36c",
-          "product_type": "ISBN",
-          "product_type_id": "bae22dfa-5ea8-43aa-b61d-8da89ba339c4",
+          "product_id": "9780547840482",
+          "product_id_type": "ISBN",
```

Comment:
This is supposed to be a UUID

Reply:
This is a type? It can be ISBN or ISSN as per Jira description

Reply:
I just picked it from Jira description example but yes it should be UUID. I have also updated the schema.

Reply:
Ahhh sorry looks like there was a miscommunication...
product_id is the id itself (not a UUID, but say an ISBN number)
product_id_type is the UUID corresponding to the type of product ID

Reply:
see https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/identifier-type.html#identifier_types_get